Cargo.toml

```diff
@@ -3,3 +3,15 @@ members = [
 	"core",
 	"registry-resolver"
 ]
+[package]
```

Comment:
This should be removed as the `ssi` package will be the one under the `core` folder. Unless there's something I'm missing about how Cargo works (could be I'm not as familiar)